src/openrct2/platform/windows.c

```diff
@@ -506,6 +506,26 @@ void platform_get_user_directory(utf8 *outPath, const utf8 *subDirectory, size_t
     }
 }
 
+bool platform_get_steam_path(utf8 *outPath, size_t outSize)
+{
+    char subkeySteam[MAX_PATH];
+    HKEY hKey;
+    DWORD type, size;
+
+    safe_strcpy(subkeySteam, "Software\\Valve\\Steam", sizeof(subkeySteam));
+
+    if (RegOpenKeyA(HKEY_CURRENT_USER, subkeySteam, &hKey) != ERROR_SUCCESS)
+        return false;
+
+    size = (DWORD)outSize;
+    LSTATUS result = RegQueryValueExA(hKey, "SteamPath", 0, &type, (LPBYTE)outPath, &size);
```

Comment:
You'll probably want to change these to use the widechar versions so that long paths and unicode paths work.

Reply:
Can you point me to the name of the functions? I found `ToUtf16` and `ToUtf8` but those are only for C++.